modules/installation-and-upgrade/pages/server-intro.adoc

```diff
@@ -4,23 +4,43 @@
 {productname} uses an [literal]``X.Y.Z`` versioning schema.
 To determine which upgrade procedure you need, look at which part of the version number is changing.
 
+[NOTE]
+====
+The version number below are just examples.
+Do not understand them as most recent available options.
+{suse} uses these numbers for illustrative purposes only!
+====
+
+
 
 Major Version Upgrade (X Upgrade)::
-* Upgrading to the next major version.
-For example, upgrading from 3.2 to 4.0 or to 4.1.
+Such a major upgrade usually is an upgrade from X.Y to X+1.0 or to X+1.1, where Y is the latest minor version of the X series.
+For example:
+
+* From version 3.2 to 4.0 or to 4.1 (upgrading directly from 3.2 to 4.2 or later is s not supported).
+* From 4.3 to 5.0 or, probably, to 5.1 (upgrading directly from 4.3 to 5.2 or later probably is not supported).
```

Comment:
We should not mention 5.X in here, especially saying probably. It's should be clear if it's supported or not. If now we don't know, we should have it in the docs.